src/utils/ar_usd_oracle.ts

```diff
@@ -0,0 +1,65 @@
+import type { FiatOracle } from './fiat_oracle';
+// import CoinGecko from 'coingecko-api';
+import type { CoinGeckoPriceRequestParams, CoinGeckoPriceResponseData, CoinGeckoResponse } from './coingecko_types';
+
+const AR_COIN_ID = 'arweave';
+const USD_COIN_ID = 'usd';
+const pollingIntervalTime = 1000 * 60 * 15; // 15min expressed in in ms
+
+export class AR_USDOracle implements FiatOracle {
+	private cachedPrice = 0;
+	private cacheTimestamp = 0;
+	private syncPromise?: Promise<void>;
+
+	// constructor(private readonly client = new CoinGecko()) {}
```

Comment:
remove unused code

Reply:
Instead of deleting it, I've ordered the comments. Those might be useful when the CoinGecko API wrapper supports browser

Reply:
Also adressed at 1986ea8139a163d1b1d73ca98b334e3405ebab99